fh-android-sdk/src/main/java/com/feedhenry/sdk2/FHRemote.java

```diff
@@ -31,12 +32,12 @@ public FHRemote(Context context) {
     }
 
     @Override
-    public void executeAsync() throws Exception {
-        executeAsync(mCallback);
+    public void execute() throws Exception {
+        execute(mCallback);
     }
 
     @Override
-    public void executeAsync(FHActCallback pCallback) throws Exception {
+    public void execute(FHActCallback pCallback) throws Exception {
         try {
             mFHHttpClient.post(getApiURl(), buildHeaders(null), getRequestArgs(), pCallback, false);
         } catch (Exception e) {
```

Comment:
@secondsun I know it isn't related of this PR, but I was thinking... Should we calling `pCallback.fail` here?